### PR TITLE
added getSystemVolume and setSystemVolume for android

### PR DESCRIPTION
**Pull request by kimxogus**
I implemented `getSystemVolume` and `setSystemVolume` in android.

### CODE REVIEW

**Comment by ofirgeller** [comment]
Hi

`public void setSystemVolume(final Integer key, final Integer value) `

You are requiring a key but not using it as the volume is not set per player.

**Comment by kimxogus** [comment]
@ofirgeller I fixed a mistake in `sound.js`, but `setSystemVolume` method in java code doesn't require a key. Please check it again.

**Comment by ofirgeller** [comment]
@kimxogus I'm commenting on the line, to my tired old eyes it seems that the method is taking an argument and then not using it.

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 1]
```diff
@@ -147,6 +149,13 @@ public void setVolume(final Integer key, final Float left, final Float right) {
   }
 
   @ReactMethod
+  public void setSystemVolume(final Integer key, final Integer value) {
```
Comment (ofirgeller) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: @kimxogus  key  is a parameter, and it is not used.

**Comment by kimxogus** [comment]
Please see the changes from all commits, not only the first commit.

You commented on the first commit I did....

You can see `Changes from 1 commit` on the top of the screenshot below.
image

**Comment by ofirgeller** [comment]
@kimxogus  my bad,  assumed the reverse order of commits then used on github.

**Comment by kimxogus** [comment]
@ofirgeller I understand, I'm sometimes confused too.

File: README.md [hunk 2]
```diff
@@ -227,6 +228,12 @@ Return the volume of the audio player (not the system-wide volume), ranging from
 ### `setVolume(value)`
 `value` {number} Set the volume, ranging from `0.0` (silence) through `1.0` (full volume).
 
+### `getSystemVolume(callback)`
```
Comment (zmxv) [inline README.md]: Please mention in the doc that it's an Android-only feature.

File: sound.js [hunk 3]
```diff
@@ -105,6 +105,25 @@ Sound.prototype.setVolume = function(value) {
   return this;
 };
 
+Sound.prototype.getSystemVolume = function(callback) {
+  if(IsAndroid) {
+    RNSound.getSystemVolume(callback);
+  } else {
+    callback({
```
Comment (zmxv) [inline sound.js]: Please remove the error callback.

**Comment by kimxogus** [comment]
@zmxv Applied your comments :)

**Comment by trepidity** [comment]
OK to merge?

**Pull request merged**